xlators/features/changelog/src/changelog.c

```diff
@@ -2566,7 +2558,16 @@ reconfigure(xlator_t *this, dict_t *options)
             if (!active_earlier) {
                 gf_smsg(this->name, GF_LOG_INFO, 0, CHANGELOG_MSG_RECONFIGURE,
                         NULL);
-                htime_create(this, priv, gf_time());
+
+                char tm_string[40];
+                snprintf(tm_string, sizeof(tm_string), "Current time : %ld",
+                         tv.tv_sec);
```

Comment:
Where do you take the time and put it into tv?

Reply:
Thanks for pointing it out.